OTCamera/hardware/button.py

```diff
@@ -192,6 +194,7 @@ def init_wifi_button():
 
     log.write("Buttons initialized")
     if external_power_button.is_pressed():
+        status.external_power_supply_connected = external_power_button.is_pressed
```

Comment:
`status.external_power_connected` will be set by `_on_external_power_button_pressed` immediatly. Do we need this line?